app/views/admin/categories/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<% provide(:title, "Lessons") %>
+<p></p>
+<h1>All lessons</h1>
+
+    <div class="container">
+
+        <table class="table table-hover">
+        <thead>
+            <tr class="bg-primary">
+                <th class="text-white" colspan="3">Category</th>
+                <th><%= link_to "Add category", new_admin_category_path, class: "btn btn-light" %></th>
+            <tr>
+            <tr>
+                <th>Title</th>
+                <th>Description</th>
+                <th>Words</th>
+                <th>Action</th>
+            </tr>
+        </thead>
+            <tbody>
+                <% @categories.each do |category| %>
+                    <tr>
+                        <td>
+                            <%= category.title %>
+                        </td>
+                        <td>
+                            <%= category.description %>
+                        </td>
+                        <td>
+                        </td>
+                        <td>
+                            <%= link_to "Edit", edit_admin_category_url(category.id) %> | <%= link_to "Delete", admin_category_path(category.id), method: :delete,
+                            data: { confirm: "You sure?"} %>
+                        </td>
+                    </tr>
+                <% end %>
+            </tbody>
+        </table>
+    </div>
```

Comment:
@shaniajubelag Fix indentions in this file.
Please observe the `.container`, `thead` and `tbody` block elements